tasks/sign-index-image/sign-index-image.yaml

```diff
@@ -46,48 +49,70 @@ spec:
       description: workspace to read and save files
   steps:
     - name: sign-index-image
-      image:
-        quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+      image: quay.io/konflux-ci/release-service-utils:7d0135b80a47cdaa225010ea1e2dff78d057c922
       script: |
-        #!/usr/bin/env sh
+        #!/usr/bin/env bash
         set -e
 
+        TASK_LABEL="internal-services.appstudio.openshift.io/group-id"
+        TASK_ID=$(context.taskRun.uid)
+        PIPELINERUN_LABEL="internal-services.appstudio.openshift.io/pipelinerun-uid"
+
         DATA_FILE="$(workspaces.data.path)/$(params.dataPath)"
         if [ ! -f "${DATA_FILE}" ] ; then
             echo "No valid data file was provided."
             exit 1
         fi
+        RPA_FILE="$(workspaces.data.path)/$(params.releasePlanAdmissionPath)"
+        if [ ! -f "${RPA_FILE}" ] ; then
+            echo "No valid rpa file was provided."
+            exit 1
+        fi
 
+        REQUESTTYPE=$(jq -r '.sign.requestType // "internal-request"' "${DATA_FILE}")
```

Comment:
What's the purpose of having two separate vars - `REQUESTTYPE` and `requestType`? Why not just set `requestType` right away?

Reply:
fixed